src/xspech.f90

```diff
@@ -243,6 +243,16 @@ subroutine read_command_args
         endif
         ext = trim(arg)
 
+        ! Prepare the "hidden" ext filepath that has a "." prefix.
+        ! Split ext into directory path and basename using INDEX function, then concatenate them again with a "." inbetween
+        #ifdef _WIN32
+          basenamestart = INDEX(ext, '\', .TRUE.)
+        #else
+          basenamestart = INDEX(ext, '/', .TRUE.)
```

Comment:
Maybe this is handled already, but what happens if there is no `/` or `\` in `ext`? Is this taken care of? According to https://gcc.gnu.org/onlinedocs/gfortran/INDEX-intrinsic.html, zero is returned if the string to be searched for is not found - but also in case of searching from the back?

Reply:
Yeah, the base case, when `input dir = cwd` and there is no `/` ext stays the same. INDEX will still return zero even for back=.true. and then we take the range `1:0` and `1:` which are both legal and what we want.